src/main/resources/assets/mmorpg/lang/zh_cn.json

```diff
@@ -1409,11 +1409,13 @@
 	"mmorpg.gear_type.cloth_helmet": "布帽",
 	"mmorpg.gear_type.cloth_pants": "布裤",
 	"mmorpg.gear_type.crossbow": "弩",
-	"mmorpg.gear_type.hammer": "战锤",
-	"mmorpg.gear_type.leather_boots": "皮靴",
-	"mmorpg.gear_type.leather_chest": "皮衣",
-	"mmorpg.gear_type.leather_helmet": "皮帽",
-	"mmorpg.gear_type.leather_pants": "皮裤",
+	"mmorpg.gear_type.dagger": "匕首",
+	"mmorpg.gear_type.gauntlets": "拳套",
+	"mmorpg.gear_type.hammer": "锤",
+	"mmorpg.gear_type.leather_boots": "皮革靴子",
+	"mmorpg.gear_type.leather_chest": "皮革上衣",
+	"mmorpg.gear_type.leather_helmet": "皮革头盔",
+	"mmorpg.gear_type.leather_pants": "皮革裤子",
```

Comment:
Change the translation of leather gears due to the consistency with the vanilla.